packages/app/src/systems/Transaction/utils/gas.test.ts

```diff
@@ -0,0 +1,20 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+import { bn } from 'fuels';
+
+import { getGasUsedFromTx } from './gas';
+
+const TX = {
+  receipts: [
+    { gasUsed: bn(10000000) },
```

Comment:
We only have one receipt with `gasUsed`, it has the receipt type `ReceiptType.ScriptResult`

Reply:
✅